lib/Graph/Graph.cpp

```diff
@@ -488,6 +502,32 @@ static void assertConvDims(NodeValue input, NodeValue filter, NodeValue bias,
   assert(bias.getType()->size() == filterDims[0] && "Invalid bias size");
 }
 
+static void assertConv3DDims(NodeValue input, NodeValue filter, NodeValue bias,
```

Comment:
Doxygen missing here too. Maybe could note it's essentially the 2D case but with the extra dimension checking.